packages/turf-center-mean/index.js

```diff
@@ -39,14 +39,15 @@ function centerMean(geojson, options) {
     var sumYs = 0;
     var sumNs = 0;
     geomEach(geojson, function (geom, featureIndex, properties) {
-        var weightValue = properties[weight] || 1;
-        if (!isNumber(weightValue)) throw new Error('weight value must be a number for feature index ' + featureIndex);
-        weightValue = Number(weightValue);
-        if (weightValue > 0) {
+        var weightValue = properties[weightTerm];
+        var weight = (weightValue === undefined || weightValue === null) ? 1 : weightValue;
+        weight = Number(weight);
```

Comment:
`weight = Number(weight)` should be after `isNumber` type checking since any invalid String given to Number() will convert it to `NaN` which will now be a number.

```js
weight = Number('foo');
//= weight => NaN
typeof weight // => 'number'
```